### DISCUSSION TITLE
VRoid SDK0.1.3 リリースのお知らせ

### DISCUSSION THREAD

**Discussion by ytomi-pixiv** [pixiv/vroid-sdk-developers#66] [Announcements]
SDK開発者のみなさま

VRoid SDK0.1.3 をリリースしました。

VRoid SDKにおいて、VRデバイス向けの新しい認可フローが利用可能になりました。

# CHANGELOG
https://vroid.notion.site/0-1-3-7d1873757d6942ae8b1a842d47be54a9

# アップデート概要

VRアプリケーションでの連携を円滑にするために、新しい認可フローを追加しました。

これにより、VRデバイス上でVRoid Hubにログインせずとも、スマートフォンなど他端末を利用することで簡単に連携できます。

詳細は以下のURLをご覧くだ さい。
https://vroid.notion.site/DeviceFlow-03e28a1676a04c1593416984a36aa33a

※VRデバイス以外のアプリケーション向けに、従来の認可フローも利用できます。

# Bug Fixed

ImageUploadのExampleで画像をアップロードしたあとUIが消えてしまう不具合を修正しました。

# 今後の対応予定

変わらず、VRM1.0モデルへの対応を最優先課題として考えております。

まずは、動作確認に使用しているUnity EditorとUniVRMのバージョンアップを行ったパッケージを近日リリース予定です。次いで、VRoid HubからのVRM1.0モデルのロード対応を行います。対応が完了次第、VRoid Hubでの配信に利用している最適化モデルをSDK経由で利用可能にするなど、モデルロードのパフォーマンス改善をリリースする予定です。

**Comment by elvneko** [comment]
最近アプリのユーザー様からVRM1.0に対する問い合わせをいただくことも増えてきたので、1.0対応とても楽しみにしております！
SDKの開発は大変なことも多いかとは思いますが影ながら応援しております。

**Comment by TwilightWings** [comment]
ちょうどVRM1.0対応を考えていましたので続報があってうれしく思います（めっちゃエラー出て気づいた）。
